src/coreclr/jit/lower.cpp

```diff
@@ -7273,13 +7273,20 @@ void Lowering::LowerIndir(GenTreeIndir* ind)
 #if defined(TARGET_ARM64)
         // Verify containment safety before creating an LEA that must be contained.
         //
-        const bool isContainable = IsSafeToContainMem(ind, ind->Addr());
+        const bool isContainable = false;
+        if (ind->Addr() != nullptr)
+        {
+            const bool isContainable = IsSafeToContainMem(ind, ind->Addr());
```

Comment:
This is hiding the other `isContainable` and then going out of scope and the value is lost.

Reply:
Yep and this is the reason for so many diffs.

Reply:
oh I forgot to remove it when moving it out of if. Thanks for spotting it.

Reply:
just a nit here - you could write it as

```
const bool isContainable = (ind->Addr() != nullptr) && IsSafeToContainMem(ind, ind->Addr());
```

if you wanted to preserve the `const`ness or shorten it.  I don't think it's a big deal.

Reply:
Thanks for the code snippet. Will make that change.